op/model.py

```diff
@@ -618,5 +616,7 @@ def storage_list(self, name):
     def storage_get(self, storage_id, attribute):
         return self._run('storage-get', '-s', storage_id, attribute, return_output=True, use_json=True)
 
-    def storage_add(self, name, count):
+    def storage_add(self, name, count=1):
+        if not isinstance(count, int) or isinstance(count, bool):
+            raise RuntimeError(f'storage count must be integer, got: {count} ({type(count)})')
```

Comment:
Could use `TypeError` here instead but we already use `RuntimeError` in similar cases in ModelBackend.

Reply:
Changed to TypeError here and created https://github.com/canonical/operator/pull/68 for other cases.